sdk/core/azure-core-management/src/main/java/com/azure/core/management/implementation/AzureProxy.java

```diff
@@ -259,111 +276,149 @@ protected Object handleResumeOperation(final HttpRequest httpRequest,
+                                            originalHttpRequest, originalHttpDecodedResponse.sourceResponse(),
+                                            delayInMilliseconds);
+                                }
+                            }
+                        } else {
+                            if (httpStatusCode == 202) {
+                                pollStrategy =
+                                    AzureAsyncOperationPollStrategy.tryToCreate(AzureProxy.this, methodParser,
+                                        originalHttpRequest, originalHttpResponse, delayInMilliseconds);
+                                if (pollStrategy == null) {
+                                    pollStrategy =
+                                        LocationPollStrategy.tryToCreate(AzureProxy.this, methodParser,
+                                            originalHttpRequest, originalHttpResponse, delayInMilliseconds);
+                                    if (pollStrategy == null) {
+                                        throw logger.logExceptionAsError(new CloudException("Response does not "
+                                            + "contain an Azure-AsyncOperation or Location header.",
+                                            originalHttpResponse));
                                     }
                                 }
+                            }
+                        }
 
-                                if (pollStrategy == null && result == null) {
-                                    pollStrategy = new CompletedPollStrategy(
-                                            new CompletedPollStrategy.CompletedPollStrategyData(AzureProxy.this, methodParser, originalHttpResponse));
-                                }
+                        if (pollStrategy == null && result == null) {
+                            pollStrategy = new CompletedPollStrategy(
+                                new CompletedPollStrategy.CompletedPollStrategyData(AzureProxy.this, methodParser,
+                                    originalHttpResponse));
+                        }
 
-                                if (pollStrategy != null) {
-                                    result = Mono.just(pollStrategy);
-                                }
+                        if (pollStrategy != null) {
+                            result = Mono.just(pollStrategy);
+                        }
 
-                                return result;
-                            });
-                });
+                        return result;
+                    });
+            });
     }
 
-    private Mono<PollStrategy> createProvisioningStateOrCompletedPollStrategy(final HttpRequest httpRequest, HttpResponse httpResponse, final SwaggerMethodParser methodParser, final long delayInMilliseconds) {
+    private Mono<PollStrategy> createProvisioningStateOrCompletedPollStrategy(final HttpRequest httpRequest,
```

Comment:
@JonathanGiles It would be nice to have guidance around this. This is really squashed to the right margin. That Oracle spec suggested to use an 8 or 4 space(?) indentation instead of aligning parameters.

Reply:
I've seen lots of patterns over my years with Java code. I will think about adding some spec guidance for this, but my inclination is to not put each argument on its own line, and to instead wrap and then indent eight spaces for all remaining arguments (or until another wrap is required).